README.md

```diff
@@ -68,24 +86,118 @@ The computation of the Shapley Values quickly becomes intensive when the number
 
+### Config file parameters
+
+#### Experiment-level parameters
+
+An experiment regroups one or several scenarios to be run.
+
+`experiment_name`: `str`  
+How the experiment will be named in output files.  
+Example: `experiment_name: my_first_experiment`
+  
+`n_repeats`: `int`  
+How many times the experiment is run.  
+Example: `n_repeats: 2`
+
+#### Scenario-level parameters
+
+##### Definition of collaborative scenarios
+
+`partners_count`: `int`  
+Number of partners in the mocked collaborative ML scenario.  
+Example: `partners_count: 4`
+
+`amounts_per_partner`: `[float]`  
+Percentages of the original dataset each partner receives to mock a collaborative ML scenario where each partner provides data for the ML training.  
+Example: `amounts_per_partner: [0.3, 0.3, 0.1, 0.3]`
+
+`samples_split_option`: `random` or `stratified`  
+How the original dataset data samples are split among partners:
+
+- `random`: the dataset is shuffled and the split is done randomly, ensuring a homogeneous data distribution among partners
+- `stratified`: the dataset is stratified per labels, so that partners have datasets covering different regions of space
+
+`corrupted_partners`: `[not_corrupted (default), shuffled or corrupted]`  
+Enables to artificially corrupt the data of one or several partners:
+
+- `not_corrupted`: data are not corrupted
+- `shuffled`: labels are shuffled randomly, not corresponding anymore with inputs
+- `corrupted`: labels are all offseted of `1` class
+  
+Example: `[not_corrupted, not_corrupted, not_corrupted, shuffled]`
+
+##### Configuration of the distributed learning approach
+
+`aggregation_weighting`: `uniform` (default), `data_volume` or `local_score`  
+After a training iteration over a given mini-batch, how individual models of each partner are aggregated:
+
+- `uniform`: simple average (non-weighted)
+- `data_volume`: average weighted with the relative amounts of data (in number of data samples)
+- `local_score`: average weighted with the performance (on a central validation set) of the individual models
+ 
+`single_partner_test_mode`: `global` (default) or `local`  
+When training a model on a single partner (this is needed in certain contributivity measurement approaches), defines if the final performance is tested on the central testset or on the partner's local testset. Note: a train-test split is performed on the original dataset, forming a central testset; this testset is also split over each partner (randomly) forming local testsets.  
+
+`epoch_count`: `int` (default: `40`)  
+Number of epochs passed as argument in the `.fit()` function. Superseded when `is_early_stopping` is set to `true`.  
+Example: `epoch_count: 30`
+
+`minibatch_count`: `int` (default: `20`)  
+Number of mini batches into which each partner's dataset is split, and over which are performed the iterations of parallel local training plus aggregation.  
+Example: `minibatch_count: 20`
+
+`is_early_stopping`: `True` (default) or `False`  
```

Comment:
when the performance **on the validation set** reaches ...

Reply:
Done